tests/webhook_test.go

```diff
@@ -1,44 +1,71 @@
 package easypost_test
 
 import (
-	"net/http"
-
 	"github.com/EasyPost/easypost-go/v2"
+	"reflect"
+	"strings"
 )
 
-func (c *ClientTests) TestWebhooks() {
+func (c *ClientTests) TestWebhookCreate() {
+	client := c.ProdClient()
```

Comment:
Please use test mode here, webhooks do not require production keys

Reply:
Nice catch!